packages/mesa.rb

```diff
@@ -39,17 +40,20 @@ class Mesa < Package
   depends_on 'wayland_protocols'
   depends_on 'zstd'
   depends_on 'lm_sensors'
+  depends_on 'ld_default' => :build
```

Comment:
Why do we need this dependency if we're not switching ld?

Reply:
We do NOT!

Reply:
(That was cruft from when I was playing around with ld_default to try to get lld to work during testing.)